api/src/main/java/org/apache/cloudstack/query/QueryService.java

```diff
@@ -92,8 +92,8 @@
     ConfigKey<Boolean> AllowUserViewDestroyedVM = new ConfigKey<>("Advanced", Boolean.class, "allow.user.view.destroyed.vm", "false",
             "Determines whether users can view their destroyed or expunging vm ", true, ConfigKey.Scope.Account);
 
-    static final ConfigKey<String> UserVMBlacklistedDetails = new ConfigKey<String>("Advanced", String.class,
-            "user.vm.blacklisted.details", "rootdisksize, cpuOvercommitRatio, memoryOvercommitRatio, Message.ReservedCapacityFreed.Flag",
+    static final ConfigKey<String> UserVMDenyListedDetails = new ConfigKey<String>("Advanced", String.class,
+            "user.vm.denylisted.details", "rootdisksize, cpuOvercommitRatio, memoryOvercommitRatio, Message.ReservedCapacityFreed.Flag",
```

Comment:
@sureshanaparti for all config keys that are renamed we need an upgrade path that update the row in cloud.configuration table for that config key (global setting).

Reply:
Updated config to 'user.vm.denied.details', upgrade path (schema file) has the relevant changes for config name change.